src/transactions/TransactionMetaFrame.cpp

```diff
@@ -201,6 +202,7 @@ void
 TransactionMetaFrame::finalizeHashes()
 {
     releaseAssert(!mHashesFinalized);
+    normalizeMeta(mTransactionMeta);
```

Comment:
we probably don't want to do this if we're not on v=3 (as this has some unknown perf impact)

Reply:
Ok.